samples/swift/uidemo/Samples/FIRStorageViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Copyright (c) 2016 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+
+import FirebaseStorageUI
+
+class FIRStorageViewController: UIViewController {
+
+  @IBOutlet private var imageView: UIImageView!
+  @IBOutlet private var textField: UITextField!
+
+  private var storageRef = FIRStorageReference().child("sample.jpg")
```

Comment:
Nah, I just didn't want to make this an optional type. But `FIRStorage.storage().reference()` makes more sense here.